src/django/api/mail.py

```
@@ -42,6 +45,23 @@ def make_claim_url(request: Request, location: Facility):
     return '{}/claim'.format(make_facility_url(request, location))
 
 
+def make_pl_search_url(request):
+    return (
+        "{}/contribute/single-location".format(
+            make_oar_url(request)
```

Ruff (0.8.2)

Undefined name `make_oar_url`

(F821)